.github/workflows/integration-tests.yml

```diff
@@ -347,3 +350,79 @@ jobs:
         continue-on-error: true
   ### End Solana Section
+
+  ### Start Live Testnet Section
+
+  testnet-smoke-tests-matrix:
+    if: startsWith(github.ref, 'refs/tags/') ## Only run live tests on new tags
+    environment: integration
+    permissions:
+      checks: write
+      pull-requests: write
+      id-token: write
+      contents: read
+    name: Live Testnet Smoke Tests Matrix
+    needs: [build-chainlink, build-test-image]
+    env:
+      SELECTED_NETWORKS: ${{ matrix.testnet }}
+      CHAINLINK_COMMIT_SHA: ${{ github.sha }}
+      CHAINLINK_ENV_USER: ${{ github.actor }}
+      TEST_TRIGGERED_BY: testnet-ci
+      TEST_LOG_LEVEL: debug
+      EVM_KEYS: ${{ secrets.QA_EVM_KEYS }}
+      TEST_EVM_KEYS: ${{ secrets.QA_EVM_KEYS }}
+      GOERLI_URLS: ${{ secrets.QA_GOERLI_URLS }}
+      TEST_GOERLI_URLS: ${{ secrets.QA_GOERLI_URLS }}
+      GOERLI_HTTP_URLS: ${{ secrets.QA_GOERLI_HTTP_URLS }}
+      TEST_GOERLI_HTTP_URLS: ${{ secrets.QA_GOERLI_HTTP_URLS }}
+      OPTIMISM_GOERLI_URLS: ${{ secrets.QA_OPTIMISM_GOERLI_URLS }}
+      TEST_OPTIMISM_GOERLI_URLS: ${{ secrets.QA_OPTIMISM_GOERLI_URLS }}
+      OPTIMISM_GOERLI_HTTP_URLS: ${{ secrets.QA_OPTIMISM_GOERLI_HTTP_URLS }}
+      TEST_OPTIMISM_GOERLI_HTTP_URLS: ${{ secrets.QA_OPTIMISM_GOERLI_HTTP_URLS }}
+      ARBITRUM_GOERLI_URLS: ${{ secrets.QA_ARBITRUM_GOERLI_URLS }}
+      TEST_ARBITRUM_GOERLI_URLS: ${{ secrets.QA_ARBITRUM_GOERLI_URLS }}
+      ARBITRUM_GOERLI_HTTP_URLS: ${{ secrets.QA_ARBITRUM_GOERLI_HTTP_URLS }}
+      TEST_ARBITRUM_GOERLI_HTTP_URLS: ${{ secrets.QA_ARBITRUM_GOERLI_HTTP_URLS }}
+    strategy:
+      fail-fast: false
+      matrix:
+        testnet: [GOERLI, OPTIMISM_GOERLI, ARBITRUM_GOERLI]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      ## Only run OCR smoke test for now
+      - name: Run Tests
+        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@e72f0a768ac934afce498a802de893d89b12802f # v2.1.1
+        env:
+          TEST_SUITE: smoke
+          TEST_ARGS: -test.timeout 30m
+          CHAINLINK_IMAGE: ${{ env.CL_ECR }}
+          CHAINLINK_VERSION: latest.${{ github.sha }}
+        with:
+          test_command_to_run: make test_need_operator_assets && cd ./integration-tests && go test -timeout 30m -count=1 -json -test.parallel=1 ./smoke/ocr_test.go 2>&1 | tee /tmp/gotest.log | gotestfmt
+          test_download_vendor_packages_command: make gomod
+          cl_repo: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink
+          cl_image_tag: latest.${{ github.sha }}
+          artifacts_location: ./integration-tests/smoke/logs
+          publish_check_name: ${{ matrix.testnet }} OCR Smoke Test Results
+          triggered_by: ${{ env.TEST_TRIGGERED_BY }}-${{ matrix.testnet }}
```

Comment:
I see you are adding the triggered_by and not specifying a `should_cleanup`, so you are expecting the cleanup logic will always clean these up. Do we want these to only clean up if the tests pass and clean themselves up or do we want them to just always be cleaned up?

Reply:
Idea is that these will have live testnet funds, and some semi-flakey return functions. Would be nice to leave them up in case of that failing and we want them back.

Reply:
I think you want to add `should_cleanup: false` to this to let it stay up in case of failure then.

Reply:
If we later on find we don't care so much, can always add this back in. Otherwise TTL should mostly take care of things.